test/e2e/fast-prototyping/tmpl-gen.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
```

Comment:
Which file is generated by this script? Or where the printed file's contents should be put?